src/core/dfn.js

```diff
@@ -71,11 +71,12 @@ export function run() {
     computeType(dfn, linkingText);
     computeExport(dfn);
 
-    // Only add `lt`s that are different from the text content
-    if (titles.length === 1 && linkingText === norm(dfn.textContent)) {
-      continue;
+    // Only add `lt`s that are different from the text content and local-lts
+    const localLt = (dfn.dataset.localLt || "").split("|");
```

Comment:
`getDfnTitles` calls `norm` on individual entries to trim spaces. Since you're going to compare strings, that seems needed as well?

 ```suggestion
    const localLt = (dfn.dataset.localLt || "").split("|").map(norm);
```